packages/@ember/-internals/glimmer/lib/helper.ts

```diff
@@ -76,11 +79,26 @@ export interface SimpleHelper<T = unknown> {
   @public
   @since 1.13.0
 */
-let Helper = FrameworkObject.extend({
+interface Helper {
+  /**
+    Override this function when writing a class-based helper.
+
+    @method compute
+    @param {Array} params The positional arguments to the helper
+    @param {Object} hash The named arguments to the helper
+    @public
+    @since 1.13.0
+  */
+  compute(params: unknown[], hash: object): unknown;
```

Comment:
This is totally reasonable and good for now; we'll probably want to revisit it once we have consensus around the Glimmer Component type changes, as we'll likely want to align roughly with that for a Glint-compatible signature here, but I don't want to try to do anything but this for *now*.